Sources/Widgets/Representations/PolyLineRepresentation/index.js

```diff
@@ -115,7 +115,12 @@ function vtkPolyLineRepresentation(publicAPI, model) {
         vtkMath.distance2BetweenPoints(last, prevLast) > model.threshold
           ? 0
           : 1;
-      if (!delta && lastState.isVisible && !lastState.isVisible()) {
+      if (
+        model.ignoreLastHiddenPoint &&
+        !delta &&
+        lastState.isVisible &&
+        !lastState.isVisible()
+      ) {
         delta++;
       }
       size -= delta;
```

Comment:
Indeed, it will be clearer like that. \
I also think it is a better practice to "reset" the moveHandle once the widget is placed, rather than just hiding it.